resources/prosody-plugins/mod_fmuc.lua

```diff
@@ -294,3 +299,30 @@ module:hook('muc-private-message', function(event)
             "Private messaging is disabled on visitor nodes"));
     return true;
 end, 10);
+
+-- we calculate the stats on the configured interval (60 seconds by default)
+module:hook_global("stats-update", function ()
+    local participants_count, rooms_count, visitors_count = 0, 0, 0;
+
+    -- iterate over all non healthcheck rooms
+    for room in prosody.hosts[module.host].modules.muc.each_room() do
+    rooms_count = rooms_count + 1;
+        if not is_healthcheck_room(room.jid) then
+            for _, o in room:each_occupant() do
+            if jid.host(o.bare_jid) == main_domain then
+                visitors_count = visitors_count + 1;
+            else
+                participants_count = participants_count + 1;
+            end
+        end
+        -- do not count jicofo
+        participants_count = participants_count - 1;
```

Comment:
Do we need the `is_healthcheck_room` here? I don't think we have health check rooms here, but if we do then participants_count will be off (unless I'm confused by the whitespace)